diff --git a/database-rules.bolt b/database-rules.bolt
deleted file mode 100644
index ea1b7d6..0000000
--- a/database-rules.bolt
+++ /dev/null
@@ -1,29 +0,0 @@
-path /canvas/the-one-and-only {
-  read() { true }
-  write() { true }
-}
-
-path /canvas/{id} is Canvas;
-
-type Canvas {
-  depth: Integer;
-  content: Square<Square<Square<Square<Square<Square<Square<Color>>>>>>>
-
-  validate() { this.depth == 7 }
-}
-
-type Square<T> {
-  '00': T | Null;
-  '01': T | Null;
-  '10': T | Null;
-  '11': T | Null;
-}
-
-type Color extends Integer {
-  validate() { this >= 0 && this < 16 }
-}
-
-// From https://github.com/firebase/bolt/blob/87a57ac87486913c50f9484b388a54fcfa61204e/samples/regexp.bolt
-type Integer extends Number {
-  validate() { (this + '').test(/^-?\d+$/) }
-}
diff --git a/package.json b/package.json
index 7f4695a..a99a9ae 100644
--- a/package.json
+++ b/package.json
@@ -3,6 +3,6 @@
     "firebase-bolt": "^0.8.4"
   },
   "scripts": {
-    "build": "firebase-bolt < database-rules.bolt > database-rules.json"
+    "build": "firebase-bolt < database-rules.bolt > .database-rules.json"
   }
 }
